CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Release History
 
 ## Unreleased: mitmproxy next
-
-
+* Fix for --allow-hosts/--ignore-hosts options in WireGuard mode (#5930)
```

Comment:
Let's add a bit more detail here. How about something like this?

- DNS resolution is now exempted from `--ignore-hosts` in WireGuard Mode.
- For plaintext traffic, `--ignore-hosts` now also takes HTTP/1 host headers into account.

Improvements welcome! 😃

Reply:
I wholeheartedly agree with your proposed changes.